CMakeLists.txt

```diff
@@ -140,15 +140,29 @@ if(BUILD_TESTS)
     ${CMAKE_CURRENT_SOURCE_DIR}/src/enclave/test/http.cpp)
   target_link_libraries(http_test PRIVATE http_parser.host)
 
-  if(NOT PBFT)
-    add_unit_test(frontend_test
+  add_unit_test(frontend_test
       ${CMAKE_CURRENT_SOURCE_DIR}/src/node/rpc/test/frontend_test.cpp)
     target_link_libraries(frontend_test PRIVATE
       ${CMAKE_THREAD_LIBS_INIT}
       evercrypt.host
       lua.host
-      secp256k1.host)
+      secp256k1.host
+    )
+    if (PBFT)
```

Comment:
No need for this in the unit test. We should only link to `libbyz.host`.

Reply:
`libbyz.host` is only created in virtual mode, so when running an SGX build it will not be available. But based on your suggestion to replace `ccf_req` with `pbft::Request` I can remove the dependency all together :)